tests/broken_options_test.cpp

```diff
@@ -208,13 +208,313 @@ TEST_CASE("Usage descriptions work")
+
+        parser.group("group2").multi_option("mopt", "some multi opt").short_name("m");
+
+        parser.group("group2")
+            .option("env-opt", "This is an option to set cool stuff.")
+            .env("ENV_OPT");
+        parser.group("group2").option("env-opt-2").env("ENV_OPT2");
+
+        parser.usage(s, false);
+
+        auto actual = s.str();
+        std::string expected =
+            R"EXPECTED(usage: app_name [-tu] --env-opt --env-opt-2 --opt --opt_long [--opt_nos] --opt_nosd [--opt_with_d] [--some_long_named_option] --mopt [command line ...]
+
+about
+
+
+group1:
+  -o [ --opt ] arg                        some opt
+  --opt_long arg                          an option with an very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very long
+                                          description
+  --opt_nos [=default value]              some opt without a short, but a
+                                          default
+  --opt_nosd arg                          some opt without short and default
+  -d [ --opt_with_d ] [=default value]    some opt with a default
+  -t [ --tog ]                            some toggle
+  -u [ --togg ]                           some other toggle
+
+group2:
+  --env-opt arg                           This is an option to set cool stuff.
+                                          Can be set using the environment
+                                          variable 'ENV_OPT'.
+  --env-opt-2 arg                         Can be set using the environment
+                                          variable 'ENV_OPT2'.
+  -m [ --mopt ] arg                       some multi opt
+  -x [ --some_long_named_option ] [=some very long default parameter for this fucking thing]
+                                          an option with an very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very very very
+                                          very very very very very long
+                                          description
+)EXPECTED";
+
+        CHECK(actual.size() == expected.size());
+
+        std::size_t tmp = 0;
+        for (unsigned long i = 0; i < actual.size(); ++i)
+        {
+            if (actual[i] != expected[i])
+            {
+                std::cout << i << " " << actual[i] << " " << expected[i] << std::endl;
+                ++tmp;
+                if (tmp > 100)
+                    break;
+            }
+        }
+
+        REQUIRE(actual == expected);
+    }
+
+    SECTION("Groups output is correct #2")
```

Comment:
Which output is correct and why does it need a second test case? Please improve the description further.